src/index.ts

```diff
@@ -0,0 +1,33 @@
+import express, { Request, Response } from 'express';
+import os from 'os';
+
+const app = express();
+const port = 3000;
+app.use(express.json());
+
+app.set('trust proxy', true);
+
+app.get('/hello', (req: Request, res: Response) => {
+    
+        const name  = req.query.name as string;
+        const greeting = name ? `Hello,  ${name}`: `Hello, World!`;
+        res.json({greeting });
+  });
+  
+app.get('/info', (req: Request, res: Response) => {
+    
+        const request_time  = new Date().toISOString();
+        const host_name = os.hostname();
+        const client_address = req.ip;
+        
+        const headers = {
+          'Accept': req.headers['accept'],
+          'User-agent': req.headers['user-agent']
```

Comment:
I need all the headers not just these two

Reply:
sure got it